docs/platform/what-is-tizen/overview.md

```diff
@@ -77,13 +77,42 @@ Compiler Toolchain
    - Neural network model quantizer.
    - Various profiles and test scripts to evaluate performance on target.
 
+### On-device & among-device AI pipelines for inference
+
+NNStreamer (https://github.com/nnstreamer/nnstreamer) is a set of GStreamer plugins that allow GStreamer developers to adopt neural network models and also lets neural network developers manage neural network pipelines and their filters easily and efficiently.
+
+- Provide neural network framework connectivities (e.g., TensorFlow, Caffe) for GStreamer streams.
+  - Efficient Streaming for AI Projects: Apply efficient and flexible stream pipeline to neural networks.
+  - Intelligent Media Filters: Use a neural network model as a media filter / converter.
+  - Composite Models: Multiple neural network models in a single stream pipeline instance.
+  - Multimodal Intelligence: Multiple sources and stream paths for neural network models.
+- Provide easy methods to construct media streams with neural network models using the de-facto-standard media stream framework, GStreamer.
+  - GStreamer Users: Use neural network models as if they are yet another media filter.
+  - Neural Network Developers: Manage media streams easily and efficiently.
+
+### On-device training & AI service personalization
+
+NTrainer is a software framework for training neural network models on embedded devices which have relatively limited resources. Rather than training whole layers of a network from scratch, NNTrainer finetunes the neural network model on a device with user data for personalization. It is based on the independent open source project NNTrainer (https://github.com/nnstreamer/nntrainer) and provides various applications.
```

Comment:
Fixed typo.
```suggestion
NNTrainer is a software framework for training neural network models on embedded devices which have relatively limited resources. Rather than training whole layers of a network from scratch, NNTrainer finetunes the neural network model on a device with user data for personalization. It is based on the independent open source project NNTrainer (https://github.com/nnstreamer/nntrainer) and provides various applications.
```